index.js

```diff
@@ -51,10 +58,10 @@ function validateResponse (fastify, opts, next) {
 }
 
 function schemaErrorsText (errors) {
-  var text = ''
-  var separator = ', '
-  for (var i = 0; i < errors.length; i++) {
-    var e = errors[i]
+  let text = ''
+  const separator = ', '
+  for (let i = 0; i < errors.length; i++) {
```

Comment:
Keep the `var` here for performance.
```suggestion
  for (var i = 0; i < errors.length; i++) {
```

Reply:
interesting :thinking: